src/config/config_impl.go

```diff
@@ -268,13 +235,12 @@ func (this *rateLimitConfigImpl) GetLimit(
 	}
 
 	if descriptor.GetLimit() != nil {
-		rateLimitKey := domain + "." + this.descriptorToKey(descriptor)
+		rateLimitKey := stats.DescriptorKey(domain, descriptor)
```

Comment:
what is the reasoning behind moving `descriptorToKey` function to stats package?

Reply:
Moved to client package (where it is used currently).